docs/streaming-kinesis-integration.md

```diff
@@ -67,6 +67,8 @@ A Kinesis stream can be set up at one of the valid Kinesis endpoints with 1 or m
 
 	- `[initial position]`: Can be either `InitialPositionInStream.TRIM_HORIZON` or `InitialPositionInStream.LATEST` (see Kinesis Checkpointing section and Amazon Kinesis API documentation for more details).
 
+	- `[credentials]`: Optional AWS credentials.
```

Comment:
it's a bit misleading to say this is optional since we still need to provide a value (None/null or Some)